pkg/odo/cli/pipelines/ui/ui.go

```diff
@@ -17,7 +27,7 @@ func EnterGitRepo() string {
 		Help:    "The GitOps repository stores your GitOps configuration files, including your Openshift Pipelines resources for driving automated deployments and builds.  Please enter a valid git repository e.g. https://github.com/example/myorg.git",
 	}
 
-	err := survey.AskOne(prompt, &gitopsUrl, survey.Required)
+	err := survey.AskOne(prompt, &gitopsUrl, validateURL(gitopsUrl))
```

Comment:
This is probably not needed...

https://github.com/rhd-gitops-example/odo/compare/pipelines_feature_dev...rhd-gitops-example:private-gitlab#diff-1f3b7e6e9e1fdf37834d6d6eb97a3452R13

This does a more comprehensive validation of the URL

Reply:
Sure Kevin, the url validation has been removed for now!